### ISSUE TITLE
Thanks!

### ISSUE THREAD

**Issue by coderschoolreview** [Roselle77/Ecommerce-Purchase#1]
## Goal of this Assignment
The goal of this assignment was to introduce you to 2 main concepts in Machine Learning: 
- Data Visualization
- Exploratory Data Analysis. 

You learn how to query and clean data using pandas library in Python, make some plots which help to understand more about data with Seaborn library.

## Things you did well:
- You can go through most of the questions, this is really great and a good start !
- You can make use of Markdown very well to note down your idea.
- You actually understand how to make use of conditions and filter the data :D (I see your notes there, it is great !)

## Things to work on:
- You should improve Python and Pandas skills. It's okay if you cannot understand clearly at the first time. Keep practicing and this should not be your problem anymore. :D 
- For the question **How many people have a credit card that expires in 2025?**.
- `ecom['CC Exp Date'].apply(exp_year) == '25'` already returns a Series of True or False. The further step is to count how many values of True, and you can do this by using `sum()` function: `sum (ecom['CC Exp Date'].apply(exp_year) == '25')`

## To sum up
- Your work is really great and i believe that you can improve much faster !!
- Keep up the spirit and keep practicing :D.
